apps/src/music/views/SoundStyle.module.scss

```diff
@@ -0,0 +1,108 @@
+@import 'color.scss';
+
+.sound {
+  &Beat {
+    &Color {
+      color: #FC55FF;
+    }
+    &Fill {
+      fill: #FC55FF;
+    }
+    &Background {
+      background-color: #89308a;
+    }
+    &Border {
+      border-color: #FC55FF;
+    }
+  }
+
+  &Bass {
+    &Color {
+      color: #00C0FF;
+    }
+    &Fill {
+      fill: #00C0FF;
+    }
+    &Background {
+      background-color: #03789e;
+    }
+    &Border {
+      border-color: #00C0FF;
+    }
+  }
+
+  &Lead {
+    &Color {
+      color: #23C347;
+    }
+    &Fill {
+      fill: #23C347;
+    }
+    &Background {
+      background-color: #14742a;
+    }
+    &Border {
+      border-color: #23C347;
+    }
+  }
+
+  &Fx {
+    &Color {
+      color: #E3CB0C;
+    }
+    &Fill {
+      fill: #E3CB0C;
+    }
+    &Background {
+      background-color: #928308;
+    }
+    &Border {
+      border-color: #E3CB0C;
+    }
+  }
+
+  &Vocal {
+    &Color {
+      color: #ff8c00;
+    }
+    &Fill {
+      fill: #ff8c00;
+    }
+    &Background {
+      background-color: #a35a02;
+    }
+    &Border {
+      border-color: #ff8c00;
+    }
+  }
+
+  &Pattern {
```

Comment:
While we're at it, thoughts on updating colors for Pattern and Chord? I remember these were chosen kind of arbitrarily, maybe worth asking @markabarrett for some colors that would work with this palette?

Reply:
Good question.  They were designed to match the colors seen in the Blockly field, but open to other ideas!